server/main.js

```diff
@@ -19,7 +19,7 @@ Meteor.startup(() => {
     })
   }
   if (process.env.QLICKER_MAIL_PASSWORD) {
```

Comment:
I think that this can be removed, since we directly set MAIL_URL as an env variable (see docker-compose.yml), we should have no need for a QLICKER_MAIL_PASSWORD variable. We don't want to hardcode mailgun.